src/styles/_footer.scss

```diff
@@ -0,0 +1,39 @@
+//параметри кінця сайту, посилання на партнерські сайти//
+
+.footer {
+  height: 220px;
+  box-shadow: 0 -1px 0 #e1e7ed;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  &__main-link {
+    display: grid;
+    grid-template-columns: repeat(3, 430px);
+    grid-template-rows: 50px;
+    text-align: center;
+  }
+}
+
+.main-link__url {
+  font-weight: bold;
+}
+
+.side-link__url {
+  display: block;
+  height: 30px;
+}
+
+.side-link__url,
+.main-link__url {
+  text-decoration: none;
+  font-size: 16px;
+  line-height: 19px;
+  font-family: "Avenir-Heavy", sans-serif;
+  color: #3d4e61;
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/76413821-d05d9d00-639e-11ea-98a8-4c9adac2d600.png)
appears in five places, move all colors to variables